packages/platforms/browser/lib/auto-instrumentation/resource-load-plugin.ts

```diff
@@ -53,8 +53,10 @@ export class ResourceLoadPlugin implements Plugin<BrowserConfiguration> {
 
         const parentContext = this.spanContextStorage.first
 
-        if (parentContext) {
-          const url = new URL(entry.name)
+        const networkRequestInfo = configuration.networkRequestCallback({ url: entry.name, type: entry.initiatorType })
```

Comment:
Does it make sense to call this if there's no `parentContext` and we won't be making a span?